src/System.IO.Compression/src/System/IO/Compression/DeflateManaged/DeflateManagedStream.cs

```diff
@@ -14,10 +14,10 @@ internal sealed partial class DeflateManagedStream : Stream
     {
         internal const int DefaultBufferSize = 8192;
 
-        private Stream _stream;
+        private Stream _stream = null!;
```

Comment:
This should be Stream? rather than being initialized to null!.  Every method is checking to make sure it's not null, via the EnsureNotDisposed helper, and having it be ? more accurately reflects the possibility that it could be null.  It'll just mean we'll need to use ! in more places.

Reply:
Yes it was needed lots of `!`, and i have seen EnsureNotDisposed() always called before use, but your point makes more sense